js/classes/Calendar.js

```diff
@@ -1095,13 +1095,23 @@ class Calendar
             if (hasLunchStarted) return true;
             dayValues.push(dayEnd);
         }
+        
+        // If non-empty inputs are passed, they should be valid (be on the day values)
+        const inputs = [dayBegin, lunchBegin, lunchEnd, dayEnd];
+        const numNonEmptyInputs = inputs.filter(Boolean).length;
+
+        if (numNonEmptyInputs!==dayValues.length)
```

Comment:
We usually add spaces before and after the operator.

```suggestion
        if (numNonEmptyInputs !== dayValues.length)
```